spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "./app/models/user"
+
+describe User do
+  context "when all attributes are correct" do
+    it "is created" do
+      user = User.create(name: "John Doe",
+                         email: "user@gmail.com")
+
+      expect(user).to be_a User
+    end
+  end
+
+  context "when name not given" do
+    it "raises an error" do
+      user = User.create(email: "user@gmail.com")
+
+      expect { user.save! }
+        .to raise_error("Validation failed: Name must be entered")
```

Comment:
Use translations in specs as well

Reply:
Duh, of course! Thanks Elle - again ignore in next PRs as I will fix this across all specs